browser/resources/settings/brave_routes.ts

```diff
@@ -156,7 +156,9 @@ export default function addBraveRoutes(r: Partial<SettingsRoutes>) {
       '[Settings] Could not move autofill route to advanced route', r)
   }
   if (loadTimeData.getBoolean('isEmailAliasesEnabled') && r.AUTOFILL) {
-    r.EMAIL_ALIASES = r.AUTOFILL.createChild('/email-aliases')
+    r.EMAIL_ALIASES = r.BASIC.createSection('/email-aliases', 'email-aliases')
```

Comment:
this should probably still be a child of `AUTOFILL` - we should add this to the `cr-view-manager` in chrome/browser/resources/settings/autofill_page/autofill_page_index.html.

See brave/browser/resources/settings/brave_overrides/appearance_page_index.ts for an example of how to do this :smile: